### DISCUSSION TITLE
CefSharp  WinForms 127.3.50 Can't Call windows Sing's  App component

### DISCUSSION THREAD

**Discussion by chendemand** [cefsharp/CefSharp#4918] [General]
CefSharp Version
 127.3.50

Operating System
Windows 10

Architecture
x64

.Net Version
.Net Framework 4.8

Implementation
WinForms

Reproduction Steps

is normal in DevTools list Call windows Sing's  App component

[ServiSign Log] Windows
CGEIDSVIAdapter.js?v=0314:100 [ServiSign Log] Chrome
CGEIDSVIAdapter.js?v=0314:100 [ServiSign Log] Echo URL: https://localhost:56306
CGEIDSVIAdapter.js?v=0314:514 recv public_key_hex 
VM450 jquery-migrate.min.js:2 JQMIGRATE: Migrate is installed, version 3.3.2

CefSharp use  in vs2019 Develop tools
Can't Run Because  no Read ServiSign

Do you know how to do it?

**Comment by amaitland** [comment]
Ask on https://www.magpcss.org/ceforum/index.php
